data/filter.tsx

```diff
@@ -12,3 +12,13 @@ export const FILTER = [
   { id: 4, title: "Clothes" },
   { id: 5, title: "Musical Instruments" },
 ];
+export const FILTER1 = [
+  {
+    id: 1,
+    title: "Funiture",
+  },
+  { id: 2, title: "Books" },
+  { id: 3, title: "Electronics" },
+  { id: 4, title: "Clothes" },
+  { id: 5, title: "Musical Instruments" },
```

Comment:
this is inconsistent with categories we have on the backend. is this just for testing?